.github/workflows/code-analysys.yml

```diff
@@ -24,7 +24,7 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    if: github.repository == 'apache/incubator-seatunnel'
+    if: null != ${{ secrets.SONARCLOUD_TOKEN }}
```

Comment:
Dose `if: github.repository == 'apache/incubator-seatunnel'` not work?

Reply:
I think it should be work https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution#example-only-run-job-for-specific-repository

Reply:
It seems that his scope includes the submitted PR, I'm not sure, but I use another way.